test/functional/p2p-segwit.py

```diff
@@ -1092,13 +1082,18 @@ def test_block_relay(self, segwit_activated):
             block4 = self.build_next_block(nVersion=4)
             block4.solve()
             self.old_node.getdataset = set()
+
             # Blocks can be requested via direct-fetch (immediately upon processing the announcement)
             # or via parallel download (with an indeterminate delay from processing the announcement)
             # so to test that a block is NOT requested, we could guess a time period to sleep for,
             # and then check. We can avoid the sleep() by taking advantage of transaction getdata's
             # being processed after block getdata's, and announce a transaction as well,
             # and then check to see if that particular getdata has been received.
-            self.old_node.announce_block(block4, use_header=False)
+            # Since 0.14, inv's will only be responded to with a getheaders, so send a header
+            # to announce this block.
+            msg = msg_headers()
+            msg.headers = [ CBlockHeader(block4) ]
```

Comment:
I prefer the space there, otherwisethetextlookstoosquishedtome, so I'll leave this (until we merge a style guide for the project that calls for removing them).